libnopegl/src/node_colorstats.c

```diff
@@ -149,6 +149,8 @@ static int setup_waveform_compute(struct colorstats_priv *s, const struct pgcraf
                                   const struct ngl_node *texture_node)
 {
     struct texture_priv *texture_priv = texture_node->priv_data;
+    texture_priv->params.usage |= NGLI_TEXTURE_USAGE_SAMPLED_BIT;
```

Comment:
Can you explain in the commit description why textures shouldn't be sampled by default?

Reply:
Commit dropped. It will have its dedicated PR.